test/functional/p2p_sendtxrcncl.py

```diff
@@ -155,36 +155,41 @@ def run_test(self):
         peer.peer_disconnect()
 
         self.log.info('SENDTXRCNCL sent to an outbound')
+        ob_peer_id = 0
         peer = self.nodes[0].add_outbound_p2p_connection(
-            SendTxrcnclReceiver(), wait_for_verack=True, p2p_idx=1, connection_type="outbound-full-relay")
+            SendTxrcnclReceiver(), wait_for_verack=True, p2p_idx=ob_peer_id, connection_type="outbound-full-relay")
         assert peer.sendtxrcncl_msg_received
         assert peer.sendtxrcncl_msg_received.initiator
         assert not peer.sendtxrcncl_msg_received.responder
         assert_equal(peer.sendtxrcncl_msg_received.version, 1)
         peer.peer_disconnect()
```

Comment:
nit: If there is no guarantee that this will disconnect the peer (due to races), it might be better to remove it, or replace it with `disconnect_p2ps`, which would also avoid having to modify the index.

Reply:
ok, I used `disconnect_p2ps` everywhere instead of `peer_disconnect`, and also  changed `p2p_idx=0` everywhere for consistency - this should also work.